editor.planx.uk/src/pages/Login.tsx

```diff
@@ -1,15 +1,51 @@
+import Box from "@mui/material/Box";
+import Link from "@mui/material/Link";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
 import React from "react";
 
+const Wrapper = styled(Box)(({ theme }) => ({
+  width: "100vw",
+  height: "100vh",
+  lineHeight: "1.333",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  background: theme.palette.primary.main,
+}));
+
+const Container = styled(Box)(() => ({
```

Comment:
We'll cover this later, but I think what you're describing here is essentially this MUI component https://mui.com/material-ui/react-container/

It might make more sense to use this, and we can maybe make some variants which describe our column layouts which can be shared as opposed to just sharing a style snippet?